modular/gater/const.go

```diff
@@ -134,9 +134,11 @@ const (
a}._link_44242_1{color:#76808f;transition:var(--transition-normal);text-decoration:underline}._link_44242_1:hover{color:var(--color-active)}._loading_1slm0_1{animation:_rotation_1slm0_1 1s infinite linear}@keyframes _rotation_1slm0_1{0%{transform:rotate(0)}to{transform:rotate(360deg)}}._wallet-item_z15a5_1{display:flex;align-items:center;justify-content:center;height:68px;background:#f5f5f5;border-radius:8px;font-weight:600;font-size:18px;line-height:27px;font-family:var(--font-poppins);cursor:pointer;transition:var(--transition-normal);position:relative}._wallet-item_z15a5_1:hover{box-shadow:0 0 0 2px var(--color-active)}._wallet-item_z15a5_1:not(:last-child){margin-bottom:16px}._wallet-item_z15a5_1._active_z15a5_22{box-shadow:0 0 0 2px var(--color-active)!important}._wallet-item_z15a5_1._disabled_z15a5_25{cursor:not-allowed;box-shadow:none}._wallet-item-icon_z15a5_29{position:absolute;left:16px;top:8px}._wallet-item-loading_z15a5_34{position:absolute;top:22px;right:24px}._container_gu0i7_1{position:fixed;pointer-events:none;width:100%;max-width:1220px;height:664px;left:50%;top:50%;transform:translate3d(calc(-50% + 47px),-50%,0);opacity:.6;filter:blur(4px);z-index:-1}._container_gu0i7_1>svg{position:absolute}@media screen and (max-width: 767px){._container_gu0i7_1{transform:translate3d(-50%,-50%,0);height:280px}._container_gu0i7_1>svg{width:auto}._container_gu0i7_1 ._cube-1_gu0i7_25{left:-39px;top:-120px;height:120px}._container_gu0i7_1 ._cube-2_gu0i7_30{right:0;top:-200px;height:88px}._container_gu0i7_1 ._cube-3_gu0i7_35{left:0;bottom:-204px}._container_gu0i7_1 ._cube-4_gu0i7_39{right:0;top:208px;height:48px}._container_gu0i7_1 ._cube-5_gu0i7_44{right:48px;top:unset;bottom:-188px}._container_gu0i7_1 ._cube-6_gu0i7_49{display:block;left:-20px;top:80px;height:48px}}._cube-1_gu0i7_25{left:0;top:0}._cube-2_gu0i7_30{right:178px;top:0}._cube-3_gu0i7_35{left:98px;bottom:51px}._cube-4_gu0i7_39{right:270px;bottom:0}._cube-5_gu0i7_44{right:0;top:236px}._cube-6_gu0i7_49{display:none}._root_1kufm_1{position:fixed;top:16px;left:50%;transform:translate(-50%);display:flex;flex-direction:column;align-items:center;z-index:var(--index-toast);gap:16px}._container_1kufm_13{display:flex;align-items:center;padding:16px 20px 16px 16px;background:#ffffff;box-shadow:0 4px 24px #00000014;border-radius:8px;word-wrap:break-word}._icon_1kufm_23{display:flex;align-items:center;justify-content:center}._description_1kufm_29{margin-left:8px;font-weight:500;font-size:14px;line-height:1.4}._footer_klz2f_1{flex-shrink:0;height:48px;color:#76808f;font-weight:400;font-size:14px;line-height:17px;display:flex;align-items:center;justify-content:center;text-align:center;position:absolute;bottom:0;left:0;right:0}._title_x9q5k_1{width:100%;max-width:403px;position:relative}._title_x9q5k_1:before{content:\"\";height:0;display:block;padding-bottom:11.91%}._title_x9q5k_1 svg{width:100%;height:100%;position:absolute;left:0;top:0}\n\n</style></head><body><div id=\"root\"></div></body></html>"
```

Comment:
the page content need gateway return ？

Reply:
Yes. this is a built-in dapp, when users use browser to access the **private universal endpoint**, we provide a SP built-in way for users to connect their wallet and sign with their wallet so that they can download/view files as long as they have the permission.  This build-in way doesn't rely any extra deployment or any 3-rd website.